src/test/java/api/loans/anonymization/LoanAnonymizationTests.java

```diff
@@ -0,0 +1,105 @@
+package api.loans.anonymization;
+
+import static api.support.http.InterfaceUrls.circulationAnonymizeLoansInTenantURL;
+import static api.support.http.InterfaceUrls.circulationAnonymizeLoansURL;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseHandler;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.junit.Before;
+
+import api.support.APITests;
+import api.support.builders.AccountBuilder;
+import api.support.builders.FeefineActionsBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+class LoanAnonymizationTests extends APITests {
```

Comment:
This doesn't appear to have any tests, is this an abstract base class for anonymization API tests?

Reply:
It is abstract, indeed